services/motion/builtin/move_request.go

```diff
@@ -68,6 +68,7 @@ type moveRequest struct {
 	seedPlan          motionplan.Plan
 	kinematicBase     kinematicbase.KinematicBase
 	obstacleDetectors map[vision.Service][]resource.Name
+	cameraToBaseCache map[resource.Name]spatialmath.Pose
```

Comment:
> The only case where this would not give us the same result is if the user were to change something in their config which would warrant a Reconfigure

Or if the camera is mounted on an arm or gantry that is mounted on the base, which is being moved by something else.